backend/app/api/game.py

```diff
@@ -84,6 +84,14 @@ async def websocket_endpoint(websocket: WebSocket, room: str):
                     await connection_manager.broadcast(room, json.dumps(state))
                 elif state['type'] == 'updateError':
                     await connection_manager.send_personal_message(json.dumps(state), websocket)
+            elif decoded['type'] == "perkActive":
+                    if state['type'] == 'oneRandomLetter':
+                        GameManager.one_random_letter(decoded['player'])
```

Comment:
using `GameManager` like this will reference the class itself, *not* an instance of the class. This could work if the methods were declared static using the `@staticmethod` decorator on the function. However, this would not work in this instance as you really want to reference the specific instance of the class which can be accessed by getting the value from the key `room_id` in the `connection_manager.connected` dictionary.

However, I actually don't think these methods are applicable to the `GameManager` class, since two of these methods (`one_random_letter`, `two_random_letters`) should just be a one-liner, calling `TileManager.draw()` with n=1, or n=2, I would suggest either changing the function name to reflect the perk which would be applicable to the `GameManager`, or just calling the `TileManager` methods inline here. Alternatively, it may be a better idea to create a `Perks` class which would hold the different types of perk and whether or not they can be accessed by the player.